doc/contributing.rst

```diff
@@ -128,8 +127,6 @@ after a reload.
 Python versions
 ---------------
 
-py3status code, including modules, should run under both python 2 and python 3.
-
 Starting with version 4.0, py3status will only run using python 3.
```

Comment:
Can you say 3.26 instead?

Reply:
Done in https://github.com/ultrabug/py3status/pull/1904/commits/81a935c52d9fb575dbdf96586ae246e80b4e64ee!